.github/workflows/actions/populate-ldap/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM osixia/openldap
```

Comment:
What's about using an official image?

Reply:
Looking at https://hub.docker.com/search?q=openldap&type=image , there is no official image and this one seems the most popular. Do you have experience with a better image?

Reply:
I'd rather use an alpine + `apk add openldap` unless bootstrapping an ldap requires too much work. 
In that case I would use an image provided by an wellknown org like "bitnami/openldap". Especially when the image has been updated few days ago (compared to osixia/openldapthat are 5 months olds).

Note: I never used one or both,

Reply:
I honestly don't know. I'm just experimenting things.

I think a solution could be something like this: https://github.com/larrycai/docker-openldap/blob/master/Dockerfile but with alpine.

Then I'm not sure what to do: one possibility is launch the ldap service and populate it with the action. But I'm not sure if the action can communicate with the service (I've tried using localhost and openlda as hostname with no luck).

The other one would be to not use the service and just set up the ldap server with the GHA. But I'm not sure the server will still be up when tha action has terminated.